@kiva/kv-loan-filters/src/sortOptions.ts

```diff
@@ -9,7 +9,7 @@ const lendToFlssSort = new Map([
 	['personalized', 'personalized'],
 	['loanAmountDesc', 'amountHighToLow'],
 	['loanAmount', 'amountLowToHigh'],
-	['amountLeft', 'amountLeft'],
+	['amountFunded', 'amountFunded'],
```

Comment:
I think FLSS is not having this sort option, so we just want to rename the dropdown label.